src/js/controller/model.js

```diff
@@ -102,7 +102,7 @@ define([
 
                     /* falls through */
                 case events.JWPLAYER_MEDIA_META:
-                    var duration = data.duration;
+                    var duration = data.duration || 0;
```

Comment:
@pajong I think it is still valid to keep the `_.isNumber` since this event could be triggered from multiple providers and theres a chances that truthy non-numerical values could potentially be passed in. As for doing the check in the providers vs the model, it should definitely be in the model, but we know the bug occurred in HTML and Flash, so made the check there as well, to avoid the possibility that a NaN duration could affect other parts of the providers. Discussed this with @robwalch and I believe we agreed that this would be the correct way forward, without putting it in every provider.